src/main/resources/assets/tconstruct/book/materials_and_you/en_us/intro/storage.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title":"Storage Chests",
+  "text": [
+    {
+      "text": "So, you want to store your extra parts in a special chest, eh? If not, too bad you have the option anyway. "
+    },
+    {
+      "text": "Part chests allow you to store an abundance of extra parts you may have. Each slot can hold a stack of 16 parts max. If you need more than that, you're doing something wrong or have very convoluted recipes.",
+      "paragraph": true
+    },
+	{
+      "text": "Modifier chests can hold any item that is used to apply a modifier or ability. If an item does not go into the chest, the item is not used for a modifier or ability.",
```

Comment:
Termonology: abilities are modifiers. You are thinking upgrade or ability. Also leaves out free/slotless modifiers, which can also go in the chest. I would just simplify to it holds items used for modifiers. If you want to be specific, say it is items used for upgrade, ability, or slotless modifiers.